src/python/pants/backend/awslambda/python/target_types.py

```diff
@@ -108,9 +118,15 @@ async def resolve_python_aws_handler(
     # we need to check if they used a file glob (`*` or `**`) that resolved to >1 file.
     if len(handler_paths.files) != 1:
         raise InvalidFieldException(
-            f"Multiple files matched for the `{field_alias}` {repr(handler_val)} for the target "
-            f"{address}, but only one file expected. Are you using a glob, rather than a file "
-            f"name?\n\nAll matching files: {list(handler_paths.files)}."
+            softwrap(
+                f"""
+                Multiple files matched for the `{field_alias}` {repr(handler_val)} for the target
+                {address}, but only one file expected.
+                Are you using a glob, rather than a file name?
```

Comment:
This was originally the same paragraph, why put on a new line?

Reply:
Ah, need to think like markdown, where an empty line starts a new paragraph.